.github/ISSSUE_TEMPLATE/01_test_runner_bug.md

```diff
@@ -0,0 +1,14 @@
+---
+name: I found a bug in the test runner
+about: >-
+  Report a bug in running tests, integration with a specific platform, or
+  any behavior of 'package:test'.
+title: ''
+labels: bug
```

Comment:
Just for clarity - you prefer not to have a `package:test` label applied here?

Reply:
Yes, I'd be fine adding a `needs-triage` label or something if we are worried about the `bug` label potentially causing it to get lost.

My idea is that the user reports the bug, and the author determines which part of the implementation is relevant.